src/views/TransactionsCategories/CategpriesPage/CategoriesPage.js

```diff
@@ -0,0 +1,62 @@
+import React, { useState, useEffect } from "react";
+import { CategoriesList } from "../CategoriesList/CategoriesList";
+import categoriesService from "../../../services/api/categoriesService";
+
+export const CategoriesPage = () => {
+  const [categories, setCategories] = useState([]);
+
+  const getCategories = async () => {
```

Comment:
can we implement loading indicator for getCategories ?

Reply:
done